Microsoft.Toolkit.Uwp.UI.Controls/TileControl/TileControl.cs

```diff
@@ -453,90 +453,73 @@ private void CreateModuloExpression(ScrollViewer scrollViewer, double imageWidth
             var compositor = _containerVisual.Compositor;
 
             // Setup the expression
-            ExpressionNode expressionX = null;
-            ExpressionNode expressionY = null;
-            ExpressionNode expressionXVal;
-            ExpressionNode expressionYVal;
+            var expressionX = compositor.CreateExpressionAnimation();
+            var expressionY = compositor.CreateExpressionAnimation();
 
             var propertySetModulo = compositor.CreatePropertySet();
-            propertySetModulo.InsertScalar(imageWidthParam, (float)imageWidth);
-            propertySetModulo.InsertScalar(offsetXParam, (float)OffsetX);
-            propertySetModulo.InsertScalar(imageHeightParam, (float)imageHeight);
-            propertySetModulo.InsertScalar(offsetYParam, (float)OffsetY);
-            propertySetModulo.InsertScalar(speedParam, (float)ParallaxSpeedRatio);
-
-            var propertySetNodeModulo = propertySetModulo.GetReference();
-
-            var imageHeightNode = propertySetNodeModulo.GetScalarProperty(imageHeightParam);
-            var imageWidthNode = propertySetNodeModulo.GetScalarProperty(imageWidthParam);
+            propertySetModulo.InsertScalar("imageHeight", (float)imageWidth);
+            propertySetModulo.InsertScalar("offsetX", (float)OffsetX);
+            propertySetModulo.InsertScalar("imageWidth", (float)imageHeight);
+            propertySetModulo.InsertScalar("offsetY", (float)OffsetY);
+            propertySetModulo.InsertScalar("speed", (float)ParallaxSpeedRatio);
+
+            expressionX.SetReferenceParameter(pParam, propertySetModulo);
+            expressionY.SetReferenceParameter(pParam, propertySetModulo);
+
+            string Thing(string common, string diemtion)
```

Comment:
Better name for this function? Check name for 2nd parameter?

Reply:
Yeah thinking on the name, hence the draft. Also thanks for spotting that.

Reply:
Spelling fix d6dde31

Reply:
Named 2ceaf28